facebookads/adobjects/helpers/adaccountmixin.py

```diff
@@ -35,7 +35,8 @@ class AgencyClientDeclaration(agencyclientdeclaration.AgencyClientDeclaration.Fi
 
     class Capabilities(object):
         bulk_account = 'BULK_ACCOUNT'
-        can_create_lookalikes_with_custom_ratio = 'CAN_CREATE_LOOKALIKES_WITH_CUSTOM_RATIO'
+        can_create_lookalikes_with_custom_ratio = ('CAN_CREATE_LOOKALIKES'
+                                                   '_WITH_CUSTOM_RATIO')
```

Comment:
I changed this line following the Google style guide for Python: [Line length](https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length)